internal/slicer/slicer_test.go

```diff
@@ -792,6 +792,68 @@ var slicerTests = []slicerTest{{
 	report: map[string]string{
 		"/dir/nested/file": "file 0644 84237a05 {test-package_myslice}",
 	},
+}, {
+	summary: "Pro archives",
```

Comment:
What is this testing? I think here we are using fake archives and we are already testing the parsing logic in `setup_test.go` so it seems like this test is redundant. The one that will add the value is the version of this test where we use the real archive.

Can you add this to `TestRealArchive`? (plus the spread test).

Reply:
Ya, removed the test. It was pointless.

I will be adding a spread test. I can add one to `TestRealArchive` as well, but I think the host will need to have pro subscription for which I guess I can leave a comment actually.

Reply:
On second thought, one probably does not need a pro token to access the release and index files of a pro archive.